src/main/java/org/reactome/release/downloaddirectory/GenerateGOAnnotationFile/GOAGeneratorUtilities.java

```diff
@@ -152,16 +168,41 @@ public static boolean isProteinBindingAnnotation(GKInstance instance) throws Exc
      * @throws Exception -- MySQL exception
      */
     public static String getStableIdentifierIdentifier(GKInstance eventInst) throws Exception {
-        GKInstance stableIdentifierInst = (GKInstance) eventInst.getAttributeValue(ReactomeJavaConstants.stableIdentifier);
-        String stableIdentifierIdentifier = stableIdentifierInst.getAttributeValue(ReactomeJavaConstants.identifier).toString();
-        return stableIdentifierIdentifier;
+        GKInstance stableIdentifierInst =
+            (GKInstance) eventInst.getAttributeValue(ReactomeJavaConstants.stableIdentifier);
+
+        return stableIdentifierInst.getAttributeValue(ReactomeJavaConstants.identifier).toString();
     }
 
     /**
-     * Finds most recent modification date for a GOA line. This is a bit of a moving target since GOA lines can be generated convergently for
-     * each type of GO annotation. Depending on if it is looking at the individual protein or whole reaction level, the date attribute
-     * may not be the most recent. If it is found that the goaLine was generated earlier but that a more recent modification date exists based on the
-     * entity that is currently being checked, then it will just update that date value in the hash associated with the line. (Yes, this is weird).
+     * Iterates through the lines in the 'goaLines' list, retrieves the date associated with that line and also adds
+     * the 'Reactome' column before adding it to the gene_association.reactome file.
+     * @throws IOException -- File writing/reading exceptions.
+     */
+    public static void outputGOAFile() throws IOException {
+
+        Path goaFilepath = Paths.get(GOA_FILENAME);
+
+        Files.deleteIfExists(goaFilepath);
+        Files.createFile(goaFilepath);
```

Comment:
Yes, but it would require to embed the file creation with the starting point for writing.  I'd rather leave it as is so the file creation is distinct in case anything gets added that precedes the current starting write point.

Reply:
I'm not sure what you mean by "starting point for writing". I think the constructor for FileWriter allows an argument to choose between append and overwrite.

Reply:
Because, the Files.write method is used here instead of a FileWriter it would need to specify the create option with the call to the Files.write method the first time it is called.  I'd rather than couple the create and write of the first lines together.

For the sake of an extra line, I'd like to keep it as it is.